README.md

```diff
@@ -153,7 +156,21 @@ def handler(event, context):
   context.iopipe.label('this-invocation-is-special')
 ```
 
-## Plugins
+### Core Agent
+
+By default, the IOpipe agent comes pre-loaded with all the bundled plugins in `iopipe.contrib.*`. If you prefer to run the agent without plugins or configure which plugins are used, you can use `IOpipeCore`:
+
+```python
+from iopipe import IOpipeCore
+
+iopipe = IOpipeCore()
```

Comment:
This example would be more helpful if it has a `plugins` setting and a comment clarifying that it means the agent will be set up with _only_ those plugins